src/pvnet_app/data/satellite.py

```diff
@@ -480,6 +480,9 @@ def process(self, ds: xr.Dataset) -> xr.Dataset:
         # recent non-nan timestamp
         ds = extend_satellite_data_with_nans(ds, t0=self.t0)
 
+        # Hot fix for https://github.com/openclimatefix/satellite-consumer/issues/63
+        ds = ds.drop_duplicates(dim="time")
```

Comment:
could you do this before interpolate_missing_satellite_timestamps , as I think thats where the error happens